azurerm/internal/services/servicefabricmesh/client/client.go

```diff
@@ -0,0 +1,19 @@
+package client
```

Comment:
I think this could just be part of the `servicefabric` service package?

Reply:
I don't believe so. While they share the same prefix, they don't share much in terms of function

Reply:
+1 - same brand name, totally different product (you run it vs managed offering) these want to be separate